java/code/src/com/redhat/rhn/common/security/acl/Access.java

```diff
@@ -214,11 +215,17 @@ public boolean aclSystemHasManagementEntitlement(Object ctx, String[] params) {
         Map map = (Map) ctx;
         Long sid = getAsLong(map.get("sid"));
         User user = (User) map.get("user");
-        Server server = SystemManager.lookupByIdAndUser(sid, user);
-        if (server == null) {
+        try {
+            Server server = SystemManager.lookupByIdAndUser(sid, user);
+            if (server == null) {
+                return false;
+            }
+            return server.hasEntitlement(EntitlementManager.MANAGEMENT);
+        }
+        catch (LookupException e) {
+            LOG.warn(e);
```

Comment:
I think we don't need to log this, because this is the expected result with nonexisting systems, and there's no further action needed from the user.

Reply:
Thank you for the review. I agree with you and I removed the logs.